sdk/keyvault/azkeys/client.go

```
@@ -619,11 +601,9 @@ func (s *startDeleteKeyPoller) Poll(ctx context.Context) (*http.Response, error)
 		// Service recognizes DeletedKey, operation is done
 		s.lastResponse = resp
 		return resp.RawResponse, nil
-	} else if err != nil {
```

Another swallowed error?

---

Fixed